protocols/bizcoin/blockchain/blkparser/blockchain.go

```diff
@@ -0,0 +1,121 @@
+// Basically adaptation from the file at https://github.com/tsileo/blkparser
+package blkparser
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+)
+
+type Blockchain struct {
+	Path        string
+	Magic       [4]byte
+	CurrentFile *os.File
+	CurrentId   uint32
+}
+
+func NewBlockchain(path string, magic [4]byte) (blockchain *Blockchain, err error) {
```

Comment:
done what I could but this is a third party library adapted by @lefkoko  so, I'm not the expert on it.